tests/raftstore/cluster.rs

```diff
@@ -318,18 +322,27 @@ impl<T: Simulator> Cluster<T> {
             let mut region = self.get_region(key);
             let region_id = region.get_id();
             let req = new_request(region_id, region.take_region_epoch().clone(), reqs.clone());
-            let resp = self.call_command_on_leader(region_id, req, timeout).unwrap();
-            if resp.get_header().has_error() {
-                if self.refresh_leader_if_needed(&resp, region_id) {
-                    warn!("seems leader changed, let's retry");
-                    continue;
-                } else if try_cnt == 1 && resp.get_header().get_error().has_stale_epoch() {
-                    warn!("seems split, let's retry");
-                    try_cnt += 1;
+            let result = self.call_command_on_leader(region_id, req, timeout);
+            match result {
+                Ok(resp) => {
+                    if resp.get_header().has_error() {
+                        if self.refresh_leader_if_needed(&resp, region_id) {
+                            warn!("seems leader changed, let's retry");
+                            continue;
+                        } else if try_cnt <= 5 && resp.get_header().get_error().has_stale_epoch() {
+                            warn!("seems split, let's retry");
+                            try_cnt += 1;
+                            continue;
+                        }
+                    }
+                    return resp;
+                }
+                Err(Error::Timeout(_)) => {
```

Comment:
@BusyJay 
Some operations are not idempotent and we can't simply retry when timeout.
we still have the issue here https://github.com/pingcap/tikv/issues/427 to be solved.

The purpose of this PR is try to fix CI of drop packet case,  based on the observation:
call command on a stale leader, or leader not elected out, request will timeout.
then our test code will unwrap on None or Err and panic. 
on both situation retry is safe

it don't solve the issue427, it just works and benefit some other tests in CI.